specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-10-03-preview/HybridCompute.json

```diff
@@ -3010,6 +3473,124 @@
       ],
       "description": "Properties for the Machine ESU profile."
     },
+    "LicenseStatus": {
+      "readOnly": true,
+      "type": "string",
+      "enum": [
+        "Unlicensed",
+        "Licensed",
+        "OOBGrace",
+        "OOTGrace",
+        "NonGenuineGrace",
+        "Notification",
+        "ExtendedGrace"
+      ],
+      "x-ms-enum": {
+        "name": "LicenseStatus",
+        "modelAsString": true
+      },
+      "description": "The license status."
+    },
+    "LicenseProfileSubscriptionStatus": {
+      "type": "string",
+      "enum": [
+        "Unknown",
+        "Enabling",
+        "Enabled",
+        "Disabled"
+      ],
+      "x-ms-enum": {
+        "name": "LicenseProfileSubscriptionStatus",
+        "modelAsString": true
+      },
+      "description": "Subscription status of the Windows Server or Windows IoT Enterprise or Product feature."
+    },
+    "ProductFeature": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Product feature name."
+        },
+        "subscriptionStatus": {
+          "$ref": "#/definitions/LicenseProfileSubscriptionStatus",
+          "description": "Indicates the current status of the product features."
+        },
+        "billingStartDate": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The timestamp in UTC when the billing starts."
+        },
+        "disenrollmentDate": {
```

Comment:
a symmetry to these property names would be nice. is the billing start date also the enrollment date, and so could be called that?

Reply:
or changing disenrollmentDate to billingStopDate?

also, how will cases where a customer enrolls/unenrolls multiple times be handled?

Reply:
Based on the discussion, added **enrollmentDate**.